cpmpy/transformations/to_cnf.py

```diff
@@ -126,6 +126,9 @@ def flat2cnf(constraints):
                 cnf += [a0sub | a1 for a0sub in subcnf]
                 continue
 
+        elif expr.name == "xor":
```

Comment:
I find this a bit of a shortcut.

More proper would be, this is a proposal, to add a 'supported_globals=frozenset()' argument to the top-level function, and then here do smth like 'if expr.is_bool() and isinstnace(expr, globalCons) and expr.name not in supported_globals: to_cnf(decompose())


it is actually part of the question where globals should be decomposed, and what trnasform they should get, but at least making this one a bit more generic would be good?

rest of pull request looks ready

Reply:
Yes I agree, this also goes hand in hand with issue #219... Here we also want to leave some constraints for the solver to encode to CNF... I'll introduce a "supported" argument for this one.